Workloads usage warning. (#77)
intel/owca: workloads/README.rst

**This software is pre-production and should not be deployed to production servers.**

+-----------------------------------------------------------------------------------------------+
| Sandbox deployment recommendation warning                                                     |
+===============================================================================================+

> | Those workloads definitions and wrapper application are not                                   |

Reviewer: ```suggestion
| These workloads definitions and wrapper applications are not                                  |
```

| meant to be run as production workloads.                                                      |
| Workloads definitions are provided only for performance evaluation purposes                   |

> | and should be run only in full-controlled and isolated sandboxed environments.                |

Reviewer: ```suggestion
| and should be run only in fully controlled and isolated sandbox environments.                 |
```

|                                                                                               |
| They were build with performance and usability in mind, with following assumption:            |
|                                                                                               | 

> | - not to be mixed with production workloads, because lack of resource and access isolation,   |

Reviewer: ```suggestion
| - not to be mixed with production workloads, because of lack of resource and access isolation, |
```

| - to be run without network isolation,                                                        |
| - run with privileged account (root account),                                                 |
| - are based on third-party docker images,                                                     |

> | - to be run be cluster (environment) owner/operator, not be cluster users,                    |

Reviewer: ```suggestion
| - to be run by cluster (environment) owner/operator, not by cluster users,                    |
```

> | - those workloads were tested only in limited specific configuration (Centos 7.5, Mesos x.x,  |

Reviewer: Provide versions, please.

Author: added

> |   Kubernetes 1.13, Docker 18.09.2)                                                            |

Reviewer: Kubernetes 1.13 [does not support](https://kubernetes.io/docs/setup/release/notes/#external-dependencies) Docker 18.09

Reviewer: 1.14 [supports](https://kubernetes.io/docs/setup/release/notes/#external-dependencies), though.

|                                                                                               |
| Possible implications:                                                                        |
|                                                                                               |

> | - some load generators, generate very resource intensive work, that can interfere with        |

Reviewer: ```suggestion
| - load generators may cause very resource intensive work, that can interfere with                |
```

|   you existing infrastructure on network, storage and compute layer                           |

> | - there are no safety mechanism  so improperly constructed command definition,                |

Reviewer: ```suggestion
| - there are no safety mechanisms so improperly constructed command definition,                |
```

|   can causes leak of private sensitive information,                                           |
|                                                                                               |

> | Please treat those workloads are reference example code. For production usage for specific    |

Reviewer: ```suggestion
| Please treat these workloads are reference code. For production usage for specific           |
```

> | Please treat these workloads are reference code. For production usage for specific            |

Reviewer 2: --> `Please treat these workloads definitions as a reference code.`

Reviewer: @felidadae [code](https://www.ldoceonline.com/dictionary/code) (see point 5) is both countable and uncountable.

> | orchestrator software and configuration, please take necessary measures recommended by        |

Reviewer 2: measures/steps

| official documentation:                                                                       |
|                                                                                               |
| - `Kubernetes <https://kubernetes.io/docs/home/>`_                                            |
| - `Mesos <https://mesos.apache.org/documentation/latest/index.html>`_                         |

> | - `Aurora documentation <http://aurora.apache.org/documentation/>`_                           |

Reviewer: ```suggestion
| - `Aurora <http://aurora.apache.org/documentation/>`_                           |
```
